xbmc/cores/DataCacheCore.cpp

```diff
@@ -19,6 +21,8 @@ CDataCacheCore::CDataCacheCore() :
   m_hasAVInfoChanges = false;
 }
 
+CDataCacheCore::~CDataCacheCore() = default;
```

Comment:
Isn't this already what happens even if this is left out?

Reply:
If I do not declare it it will not be virtual, if I write `= default`in the header compiler will exit with an error because it will inline the method and cannot resolve forward declared type `EDL::Cut`.

Reply:
I see